wdl-lint/src/rules/blank_lines_between_elements.rs

```diff
@@ -557,6 +577,30 @@ fn check_prior_spacing(
     }
 }
 
+/// Check that the node's last token does not have a blank before it.
+fn check_last_token(
+    syntax: &SyntaxNode,
+    state: &mut Diagnostics,
+    exceptable_nodes: &Option<&'static [SyntaxKind]>,
+) {
+    let prev = syntax
+        .last_token()
+        .expect("Node should have last token")
+        .prev_sibling_or_token();
+    if let Some(prev) = prev {
+        if prev.kind() == SyntaxKind::Whitespace {
+            let count = prev.to_string().chars().filter(|c| *c == '\n').count();
```

Comment:
With `prev` now a `SyntaxToken` with the above change, we can use `text` here:

```suggestion
            let count = prev.text().chars().filter(|c| *c == '\n').count();
```

That saves us a string allocation; we should probably edit other places in this file (and others in `wdl-lint` that we could use `into_token` with `text` to save on a string allocation.